daml-lf/spec/daml-lf-1.rst

```diff
@@ -558,10 +558,10 @@ strings as *package identifiers*.  ::
            pversion ::= PackageVersionString        -- PackageVersion
 
   V0 Contract identifiers:
-          cidV0  ∈  #[a-zA-Z0-9\._:-#/ ]+           -- V0ContractId
+          cidV0  ∈  #[a-zA-Z0-9\._:-#/ ]{1,254}     -- V0ContractId
```

Comment:
This length limitation hasn't been enforced for a while, neither does the implementation do so at the moment. I don't know whether there are any ledgers out there that produced longer contract IDs that might be stored somewhere.

Reply:
The limitation is enforced in the code, See the definition of [ContractIdString](https://github.com/digital-asset/daml/blob/a79377cb2d6eb488314c9468d8653b621fcb1d09/daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/IdString.scala#L275)

Previously V0 was a `LegerString`, `LegerString` was always limited to 255 since its introduction last year in #1192.

Reply:
Thanks for clarifying.